kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -152,17 +212,35 @@ define([
                     lengthChange: false,
                     columns: [
                         {
-                            className: `${cssBaseClass}__cell--object`,
+                            className: `${cssBaseClass}__cell--import-type`,
                             render: (data, type, row) => {
-                                const jobInfo = jobManager.model.getItem('exec.jobs.info') || {};
+                                const jobParams =
+                                    jobManager.model.getItem('exec.jobs.paramDisplayData') || {};
+                                const relevantJobId =
+                                    row.retry_parent && jobParams[row.retry_parent]
+                                        ? row.retry_parent
+                                        : row.job_id;
 
-                                if (row.retry_parent && jobInfo[row.retry_parent]) {
-                                    return renderParams(jobInfo[row.retry_parent]);
+                                if (jobParams[relevantJobId]) {
+                                    return displayAppType(jobParams[relevantJobId].analysisType);
                                 }
-                                if (jobInfo[row.job_id]) {
-                                    return renderParams(jobInfo[row.job_id]);
+                                return `Job ID: ${row.job_id}`;
```

Comment:
show the job ID in the leftmost row if the import type has not yet been established